README.md

```diff
@@ -69,6 +69,19 @@ Below is a list of environment variables that will affect the vignette runtime.
  * `CONVERT_CONSTRAINTS`          universal options to pass to ImageMagick. see bin/thumbnail
  * `UNSUPPORTED_REDIRECT_HOST`    on an unsupported legacy thumbnail request, host to redirect
 
+## Command Line Options
+
+To see the command line options available execute `lein run -- -h`. The notable
+command line options are the following:
+
+ * -C,--cache-thumbnails: enable thumbnail caching. When this option is provided
+   thumbnails will be written to and read from the backing storage provided. The
+   default is false.
```

Comment:
Err, sorry no but the default is not specified in the `cli-specs` - does an absent CLI option default to `false`?